Src/Plugins/Security/AzureDevOpsPersonalAccessTokenValidator.cs

```diff
@@ -93,16 +93,21 @@ internal static class AzureDevOpsPersonalAccessTokenValidator
         /// <param name="failureLevel">
         /// The current failure level associated with the match (if a match occurs). This parameter can be
         /// set to a different failure level by the callee, if appropriate.
+        /// </param
+        /// <param name="fingerprint">
+        /// The matched pattern in the fingerprint format.
         /// </param>
         /// <returns>Return the validation state.</returns>
 #pragma warning disable IDE0060 // Remove unused parameter
         public static string IsValid(
             ref string matchedPattern,
             ref Dictionary<string, string> groups,
             ref bool performDynamicValidation,
-            ref string failureLevel)
+            ref string failureLevel,
+            ref string fingerprint)
```

Comment:
>ref string fingerprint [](start = 12, length = 22)

We were missing the fingerprint (fixed)

Reply:
i see the fingerprints generated in result with this change, do we intended to show them?
e.g. 
                    "fingerprints": {
                        "SecretFingerprint/v1": "[pat/vs=h5lxeqkz4zujkyvzg5emkejgelxj4x64tvyzclprekuloawbbz5q]"
                    }

---
In reply to: [553894953](https://github.com/microsoft/sarif-pattern-matcher/pull/91#discussion_r553894953) [](ancestors = 553894953)

Reply:
yes, that is by design.

---
In reply to: [554117253](https://github.com/microsoft/sarif-pattern-matcher/pull/91#discussion_r554117253) [](ancestors = 554117253,553894953)